src/tests/nativeaot/SmokeTests/UnitTests/Generics.cs

```diff
@@ -53,7 +53,10 @@ internal static int Run()
         TestNullableCasting.Run();
         TestVariantCasting.Run();
         TestMDArrayAddressMethod.Run();
-        TestNativeLayoutGeneration.Run();
+// SIGSEGV handler missing: https://github.com/dotnet/runtime/issues/87458
```

Comment:
I am reasonably sure the SIGSEGV handler is not missing. I tried to debug the app in Xcode and it happily goes over multiple SIGSEGVs. Maybe just reference the issue without being too specific?

Reply:
Sounds good, I will reference the issue without additional comments.